dask_ml/preprocessing/_block_transformer.py

```diff
@@ -0,0 +1,63 @@
+import dask.array as da
+import dask.dataframe as dd
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+
+from dask_ml.utils import check_array, handle_zeros_in_scale
+
+
+class BlockTransformer(BaseEstimator, TransformerMixin):
+    """Construct a transformer from a an arbitrary callable
+
+    The BlockTransformer forwards the blocks of the X arguments to a user-defined
+    function or function-object an returns the result of this operation.
+    This is useful for stateless operations, that can be performed on the cell or
+    block level, such as taking the log of frequencies. In general the transformer
+    is not suitable for e.g. standardization tasks as this requires information for
+    a complete columns
+
+    Parameters
+    ----------
+    func : callable
+        The callable to use for the transformation.
+
+    validate : bool, optional default=True
```

Comment:
scikit-learn is switching to `validate=False` by default. I think we should do the same. https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.FunctionTransformer.html